packages/hooks/src/useCreateNFT.ts

```diff
@@ -4,59 +4,64 @@ import { useCallback, useContext, useState } from 'react'
 import invariant from 'ts-invariant'
 import { LiteflowContext } from './context'
 import { ErrorMessages } from './errorMessages'
-import { Standard } from './graphql'
 import useCheckOwnership from './useCheckOwnership'
 import useIPFSUploader from './useIPFSUploader'
 import { convertTx } from './utils/transaction'
 
 gql`
-  mutation CreateAsset(
-    $input: AssetInputBis!
-    $creatorAddress: Address!
-    $amount: Uint256!
+  mutation CreateAssetTransaction(
+    $chainId: Int!
+    $collection: Address!
+    $metadata: Metadata!
+    $creator: Address!
```

Comment:
Would be good to keep the naming consistent with API:
```suggestion
    $creatorAddress: Address!
```